.gitignore

```diff
@@ -9,11 +9,9 @@ Cargo.lock
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
-# Other
-.DS_Store
-
-# VSCode project
+# IDE/Editor configuration files
```

Comment:
Could we keep `.DS_Store` here please?

Reply:
Ahhh accident sorry. I’ll add it back in the morning